Dockerfile

```diff
@@ -1,11 +1,25 @@
-FROM python:3.11-slim-buster
+FROM python:slim
+
+LABEL org.opencontainers.image.source="https://github.com/toddrob99/searcharr"
+LABEL org.opencontainers.image.description="Docker for SEARCHARR"
+LABEL Name=Searcharr Version="v1.2-beta"
+
+ARG TARGETPLATFORM BUILDPLATFORM
 
-LABEL Name=Searcharr Version=1.2
 
 WORKDIR /app
+
+RUN chmod -R 777 /app && \
+    chmod -R +x /app && \
+    chmod -R 705 /app
```

Comment:
Why?

Reply:
gives full permissions to the /app directory in case of any ``` Permissions Denied``` errors

Reply:
Did you encounter any Permission Denied errors? I have not, and I have not received any reports of such issues. This seems unnecessary.